docs/DeveloperGuide.md

```diff
@@ -256,47 +256,67 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 Priority | As a ... | I want to ... | So that I can...
```

Comment:
@chanjunweimy Thanks for the feedback! Diagrams in Developer Guide are now updated :) !